test/active_model_serializers/test/serializer_test.rb

```diff
@@ -0,0 +1,71 @@
+require 'test_helper'
+require 'rails-controller-testing' if Rails::VERSION::MAJOR >= 5
```

Comment:
[This test uses](https://github.com/maurogeorge/active_model_serializers/blob/patch-02/test/active_model_serializers/test/serializer_test.rb#L67), this test was based on the original codebase of the test helper.